shader_source.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+
+import os
+import requests
+
+
+@dataclass
+class GetShaderCode:
+    shade_id: str
+    api_key: str = os.environ['API_KEY']
+
+    def _request_shade(self):
```

Comment:
rename `_request_shader` and take `shader_id` as a parameter

Reply:
:facepalm: sure, that make way more sense than creating a new object everytime